cli/src/commands/runTests.js

```diff
@@ -46,9 +46,7 @@ type TestGroup = {
  * structs. Each TestGroup represents a Package/PackageVersion/FlowVersion
  * directory.
  */
-const basePathRegex = new RegExp(
-  `definitions${path.sep}npm${path.sep}(\@[^${path.sep}]*${path.sep})?[^${path.sep}]*${path.sep}?`,
-);
+const basePathRegex = new RegExp('definitions/npm/(@[^/]*/)?[^/]*/?');
```

Comment:
@GAntoine Does this still work with `\` as path separator? I think the checks for `^/` will miss a backslash on Windows as well as the terminating optional `/`.
The correct fix should be to use `path.sep`, but to double it in case it is a backslash (correct Regex escape).

E.g.
```
...\flow-typed\definitions\npm\mylib\
```
would not match

Reply:
It no longer works with a backslash, but according to

> Note: On Windows, both the forward slash (/) and backward slash (\\) are accepted as path segment separators;

that shouldn't be an issue.

Reply:
Yes, Windows works with forward slashes for all fs operations, but my point was, that the regex is used to _parse_ incoming paths, which still have backslashes embedded on Windows, so the forward slash only regex is unsuitable to parse Windows paths.

Reply:
Oh I see, I wasn't aware of that... Would you mind opening a new issue? I'll ping the right people and we can discuss it there. I don't work on windows so I can't really verify either change, more deeply then how sound they look.